test/scenarios/synthetic/monolocation/suite.go

```diff
@@ -0,0 +1,273 @@
+//go:build e2e
+
+package monolocation
```

Comment:
what do you mean by `monolocation`?

Reply:
I was searching a neologism for the scenario which evaluates a location with a single VUC. Intended to create another for a location with a pair of VUCs (_stereoloc_). For now, postponed the latter.